pkg/kubernetes/kubernetes.js

```diff
@@ -147,6 +147,21 @@ define([
             }).fail(failure);
         };
 
+        this.create_replicationController = function create_replicationController(replicationController_json) {
+            api.post("/api/v1beta1/replicationControllers", JSON.parse(JSON.stringify(replicationController_json)))
```

Comment:
I would suggest that the JSON passed to this API should be JSON ready for use, and shouldn't need the JSON.parse(JSON.stringify(...)) normalization.